sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2476,22 +2456,6 @@ class DataSourceV2SQLSuite
       .head().getString(1) === expectedComment)
   }
 
-  test("SPARK-30799: temp view name can't contain catalog name") {
-    val sessionCatalogName = CatalogManager.SESSION_CATALOG_NAME
-    withTempView("v") {
-      spark.range(10).createTempView("v")
-      val e1 = intercept[AnalysisException](
-        sql(s"CACHE TABLE $sessionCatalogName.v")
```

Comment:
This is already tested in https://github.com/apache/spark/pull/34490/files#diff-eeb429a8e3eb55228451c8dbc2fccca044836be608d62e9166561b005030c940R2317